libr/core/cmd_anal.c

```diff
@@ -1622,6 +1622,21 @@ static void core_anal_bytes(RCore *core, const ut8 *buf, int len, int nops, int
 				r_anal_esil_stack_free (esil);
 			}
 		} else {
+			char disasm[128] = { 0 };
+			r_parse_varsub (core->parser, NULL,
+				core->offset + idx,
+				asmop.size, r_asm_op_get_asm (&asmop),
+				disasm, sizeof (disasm));
+				ut64 killme = UT64_MAX;
+				if (r_io_read_i (core->io, op.ptr, &killme, op.refptr, be)) {
```

Comment:
warning here, you should read specifying the endiannes , otherwise this wont be portable.

Reply:
Not sure how to deal with endianness here. I thought it was being assumed that it was big endian hence the use of `be`. Any advice?